rust/src/conf.rs

```diff
@@ -46,27 +50,33 @@ impl ClientOption {
     pub fn access_url(&self) -> &str {
         &self.access_url
     }
+    /// set the access url of RocketMQ proxy
```

Comment:
Please follow rust documentation examples to include arguments
https://doc.rust-lang.org/rust-by-example/meta/doc.html